analog-asteroid-logo/usr/share/asteroid-launcher/watchfaces-img/asteroid-logo.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg width="2000" height="2000" shape-rendering="crispEdges" xmlns="http://www.w3.org/2000/svg"><defs><rect id="T" width="2" height="2"/></defs><g transform="matrix(100 100 -100 100 1000 0)"><g id="row1"><use href="#T" id="a" style="fill:#be3729"/><use href="#T" id="b" x="2" style="fill:#dc2919"/><use href="#T" id="c" x="4" style="fill:#e54b3a"/><use href="#T" id="d" x="6" style="fill:#e56934"/><use href="#T" id="e" x="8" style="fill:#e57c21"/></g><g id="row2" transform="translate(-2,2)"><use href="#b"/><use href="#c"/><use href="#T" id="f" x="10" style="fill:#e58a21"/></g><g id="row3" transform="translate(-4,4)"><use href="#c"/><use href="#e"/><use href="#T" id="g" x="12" style="fill:#f19a11"/></g><g id="row4" transform="translate(-6,6)"><use href="#d"/><use href="#e"/><use href="#f"/><use href="#T" id="h" x="14" style="fill:#f0ae0e"/></g><g id="row5" transform="translate(-8,8)"><use href="#e"/><use href="#f"/><use href="#g"/><use href="#h"/><use href="#T" x="16" style="fill:#f0c30e"/></g></g></svg>
```

Comment:
Tested `shape-rendering="crispEdges"` on device in all watchfaces containing the logo and the newly added splash transition animation. Found no problems. To the contrary, the logo appears sharper in many situations.
This solves the "transparent lines between shapes" glitch mentioned earlier.